### PR TITLE
[Brainbrowser] file_url query param

### PR DESCRIPTION
**Pull request by laemtl**
Based on #8643

Support for an additional query parameter to pass file urls: 
```
brainbrowser/?minc_id=32,33&file_url=https://loris.ca/api/v0.0.3/candidates/867710/V1/images/inhance_867710_V1_F18FEOBVOSEM_001.mnc,https://loris.ca/api/v0.0.3/candidates/867710/V1/images/inhance_867710_V1_F18FEOBVOSEM_002.mnc
```
This is useful from some particular projects where files are stored on an external server (no files db).
minc_id and file_url can be both supplied with values and the result will be merged in the response.

### CODE REVIEW

File: modules/brainbrowser/js/brainbrowser.loris.js [hunk 1]
```diff
@@ -1,12 +1,17 @@
...
-    "use strict";
-    var query = window.location.search.substring(1),
-        vars = query.split("&"),
-        i,
-        pair;
+    'use strict';
+    let query = window.location.search.substring(1);
+        let vars = query.split('&');
+        let i;
+        let pair;
```
Comment (xlecours) [inline modules/brainbrowser/js/brainbrowser.loris.js:12]: 1 too many level of indentation here

**Review by xlecours — APPROVED** [review]
I have to test it now :)

**Comment by laemtl** [comment]
@xlecours @driusan Reviewing this I believe the fix to display errors should be part of the release since it may fix all the reported issues for the modules: #8016, #7681

**Comment by driusan** [comment]
@laemtl This can't be merged because of conflicts

**Comment by laemtl** [comment]
@driusan Ready

**Pull request merged**